### DISCUSSION TITLE
Why only counts contributions when I made 'git push --force origin master'

### DISCUSSION THREAD

**Discussion by julianddress** [community/community#67783] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I tried to complete my commits with 'git push origin <my-branch>' but when I realized they did not count for my contributions just only when I do it with git push --force origin a <my-branch>. Why this happens and how can I fix it :(

**Comment by Abhiram-J** [comment]
Hi julianddress,
The difference between 'git push ' and 'git push --force ' is that the latter overwrites the files which had merge conflicts. If there is any conflicts then git push will mention that, but when --force flag is added it would forcefully push into github overwriting the conflicted files in remote with the local files.

To know about how contributions work click [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)
